Tests/FluentMySQLTests/FluentMySQLTests.swift

```diff
@@ -206,6 +206,59 @@ class FluentMySQLTests: XCTestCase {
         test = try test.save(on: conn).wait()
     }
 
+    func testReferences() throws {
+        let conn = try benchmarker.pool.requestConnection().wait()
+        defer { benchmarker.pool.releaseConnection(conn) }
+
+        // Prep tables
+        defer { try? Child.revert(on: conn).wait()
+                try? Parent.revert(on: conn).wait() }
```

Comment:
formatting should be done by increments of tabs. vapor repos try to avoid the "xcode style" of lining things up to make it easier to contribute from non-xcode ides